extensions/wikia/AdEngine/js/wad/btRecLoader.js

```diff
@@ -14,37 +14,53 @@ define('ext.wikia.adEngine.wad.btRecLoader', [
 		placementsMap = {
 			TOP_LEADERBOARD: {
 				uid: '5b33d3584c-188',
-				style: 'margin: 10px 0;'
+				style: 'margin:10px 0; z-index:100;',
+				lazy: false
 			},
 			TOP_RIGHT_BOXAD: {
 				uid: '5b2d1649b2-188',
-				style: 'margin-bottom: 10px;'
+				style: 'margin-bottom:10px; z-index:100;',
+				lazy: false
+			},
+			BOTTOM_LEADERBOARD: {
+				uid: '5b8f13805d-188',
+				style: 'margin-bottom:23px; z-index:100;',
+				lazy: true
 			}
 		};
 
-	function markAdSlots(replace) {
+	function markAdSlots(replace, lazySlotName) {
```

Comment:
If I understand correctly this method marks ad slots, replaces the one to be recovered and executes vendor's function depending on the 2nd parameter passed to it. It's quite a lot of duties for such a small method ;) What do you think of splitting the responsibilities a little bit? :)

Reply:
That's a good observation. I refactored the code a little bit.